tests/gemm/test_dense_gemm_expected_m.py

```
@@ -18,16 +19,194 @@
+            sf_vec_size=32,
+            sm_count=SM,
+            mma_tiler_mn=(64, 64),
+            load_path="tma",
+            swap_ab=False,
+        )
+
+
+def test_grouped_output_rejects_swapped_storage() -> None:
+    lhs, rhs = _unaligned_dense_operands(1, 132, 128, 2)
+
+    with pytest.raises(ValueError, match="swapped dense_gemm.*supports L=1 only"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)